app/backend/gwells/static/gwells/css/main.css

```diff
@@ -2300,7 +2300,7 @@ h5.twitterUserName a {
 #main-content #body a {
     word-wrap: break-word;
     /* RA-636 - Non standard for webkit */
-    word-break: break-all;
+    /* word-break: break-all; */
```

Comment:
I'd prefer you remove the line entirely. Git will always have a record of what it used to be, we don't need dead code bloating the file. Also - the comment above it is also rendered redundant and you could remove that at the same time.

Reply:
Done and squashed!